mm/mixtures.py

```diff
@@ -117,6 +117,7 @@ def merge_helper(elements):
             util.partition_into_tree(self.mixtures_to_combine, self.combine_per_step),
         )
         shutil.move(mixtures, self.out_mixtures.get_path())
+        os.chmod(self.out_mixtures.get_path, stat.S_IROTH)
```

Comment:
```suggestion
        os.chmod(self.out_mixtures.get_path(), stat.S_IROTH)
```

Call the function.

Reply:
Thanks for the reply. This should be done now with the new commit. You were right with the setting of the mode, so I did it bitwise instead as suggested.